### DISCUSSION TITLE
loadSVGFromURL needs to verify the cookie

### DISCUSSION THREAD

**Discussion by sanheng** [fabricjs/fabric.js#6977] [Q&A]
Obtaining svg through fabric.loadSVGFromURL() needs to verify the cookie, but we are cross-domain and need withCredentials, what should I do？

**Comment by asturur** [comment]
So LoadSVGFromURL accpet a crossOrigin option but that is used only to load the images inside the SVG.
You should request the SVG from the URL with your on code and then use loadSVGFromString to parse it.

**Reply by sanheng** [reply]
Thanks, let me try
